demo/midgard-sdk/src/linked-list.ts

```diff
@@ -60,3 +108,77 @@ export const getNodeDatumFromUTxO = (
 export class LinkedListError extends EffectData.TaggedError(
   "LinkedListError",
 )<GenericErrorFields> {}
+
+export type LinkedListInitParams = {
+  policyId: string;
+  address: string;
+  mintScript: Script;
+  dataSchema: any;
+  data: any;
```

Comment:
Using `any` is code smell. Either use type parameters, or take a different approach (e.g. just get the `Data` or its CBOR hex).